### PR TITLE
commitizen 2.4.1 (new formula)

### LINKED ISSUE #256: Make commitizen homebrew installable
## Description
Make commitizen homebrew installable

## Possible Solution
Publish to homebrew

### PR DESCRIPTION
**Pull request by harens**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Closes https://github.com/commitizen-tools/commitizen/issues/256

### CODE REVIEW

File: Formula/commitizen.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    # Generates a changelog after an example commit
+    system "git", "init"
+    touch "example"
+    system "git", "add", "example"
+    system "yes | cz commit"
```
Comment (dawidd6) [inline Formula/commitizen.rb:87]:
```suggestion
    system "yes | #{bin}/cz commit"
```

File: Formula/commitizen.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    system "cz", "changelog"
```
Comment (dawidd6) [inline Formula/commitizen.rb:88]:
```suggestion
    system "#{bin}/cz", "changelog"
```

File: Formula/commitizen.rb (hunk 1 above)
Comment (harens) [reply Formula/commitizen.rb:87]: Thanks for your suggestions @dawidd6.

Comment (harens) [reply Formula/commitizen.rb]: I've squashed the commits and left you as a co-author. Thanks again for your help.

**Review by dawidd6 — APPROVED** [review]

**Comment by issyl0** [comment]
Thanks for your contribution to Homebrew, @harens!

**Pull request closed without merging**